src/vision/how_to_vision/shiny_future.md

```diff
@@ -1,8 +1,6 @@
 # ❓ How to vision: "Shiny future" stories
 
-## 🛑 Not time for this yet 🛑
-
-We're not ready for this yet! See the [how to vision](../how_to_vision.md) page for details on the phasing.
+We want all Async Rust users and their hopes and dreams for what Async Rust should be in the future to be reflected in the async vision doc, so please help us by writing 'shiny future' stories about what you would like async Rust to look like! Remember, **shiny future visions can be personal and may differ greatly from person to person. Therefore it's impossible for these visions to be wrong.** 
```

Comment:
```suggestion
We want all Async Rust users and their hopes and dreams for what Async Rust should be in the future to be reflected in the async vision doc, so please help us by writing 'shiny future' stories about what you would like async Rust to look like! **Remember: we are in a brainstorming period.** Please feel free to leave comments in an effort to help someone improve their PRs, but if you would prefer a different approach, you are better off writing your own story. (In fact, you should write your own story even if you like their approach but just have a few alternatives that are worth thinking over.)
```

Maybe like this?

Reply:
That seems better, I was also thinking that shiny future visions could be a little more "wrong" than status quo stories.

Reply:
I also removed the parenthesis in the sentence you mentioned. Do you want parenthesis removed else where as well?

Reply:
Oh, I misunderstood what you meant, actually removed the parenthetical now :sweat_smile: